libraries/AP_HAL_ChibiOS/hwdef/AET-H743-Basic/README.md

```diff
@@ -0,0 +1,126 @@
+# AET-H743-Basic Flight Controller
+
+The AET-H743-Basic is a flight controller designed and produced by AeroEggTech
+
+## Features
+
+ - STM32H743 microcontroller
+ - Dual ICM42688P IMUs
+ - 13 PWM / Dshot outputs
+ - 8 SERIALs
+ - 1 CAN
+ - Dedicated USB board
+ - DPS310 or SPL06 barometer
+ - 5V/6V/7V 10A BEC; 9V 2A BEC; 5V 2A BEC
+ - MicroSD Card Slot
+ - 2-way camera input
+ - AT7456E OSD
+ - 2 I2Cs
+
+## Physical
+
+![AET-H743-Basic overview](AET-H743-Basic_overview.jpg)
+
+![AET-H743-Basic core board](AET-H743-Basic_core_board.png)
+
+![AET-H743-Basic power board](AET-H743-Basic_power_board.png)
+
+
+## Mechanical
+
+ - Dimensions: 36 x 47 x 17 mm
+ - Weight: 45g
+
+
+## Power supply
+
+The AET-H743-Basic supports 2-6s Li battery input. It has 3 ways of BEC, which result in 6 ways of power supplys. Please see the table below.
+
+| Power symbol | Power source | Max power (current) |
+|--------------|--------------|---------------------|
+| 5V | from 5V BEC | 20W (4A) |
+| 9V | from 9V BEC | 18W (2A) |
+| 9Vsw | from 9V BEC, controlled by MCU with an NPN MOS | 10W (1A) |
+| 4V5 | from USB or 5V BEC, with a diode | 5W (1A) |
+| VX | from VX BEC, default 5V, can be changed to 6V or 7V | 50W (10A) |
+| BAT | directly from battery | (5A) |
+
+
+## Loading Firmware
+
+Initial firmware load can be done with DFU by plugging in USB with the bootloader button pressed. Then you should load the "with_bl.hex" firmware, using your favorite DFU loading tool, such as Mission Planner.
+
+Once the initial firmware is loaded you can update the firmware using any ArduPilot ground station software. Updates should be done with the "\*.apj" firmware files.
+
+## UART Mapping
+

```

Comment:
All UARTS are DMA capable except TBD

TBD is depending on if another UART like UART2 is moved to NOMDA

Reply:
UARTs need to state if they have DMA or not...please either add the "All UARTs...." statement or mark each UART line with DMA capable were applicable

Reply:
Done